tests/TestCase/TestSuite/IntegrationTestCaseTest.php

```diff
@@ -43,7 +44,8 @@ public function setUp()
         static::setAppNamespace();
 
         Router::reload();
-        Router::scope('/', function ($routes) {
+        Router::extensions(['json']);
```

Comment:
Because routes are added in the setup method I don't think the router would be reloaded as routes would not be automatically loaded.

Reply:
Do you want to change anything on that test case?
Feel free to add a commit on top with desired changes.

Reply:
It'll be ok. I think this code won't be needed in 4.x, or at least I hope it isn't.